client/app/assets/less/inc/base.less

```diff
@@ -26,7 +26,11 @@ body {
     
     &.headless {
         padding-top: 10px;
-        
+
+        app-view {
+          min-height: ~"calc(100vh - 10px)";
```

Comment:
I think this would be a simpler solution:

```less
&.headless {
  padding-top: 10px; // <- remove this line

  app-view {
     padding-top: 10px; // <- move it to here
  }
```

Reply:
Fixed.